index.html

```diff
@@ -231,6 +231,6 @@ <h2 class="success__title">Изображение успешно загруже
   <template id="messages">
     <div class="img-upload__message  img-upload__message--loading">Загружаем...</div>
   </template>
-
+<script src = "js/main.js"></script>
```

Comment:
Тут пробелы не надо ставить
```suggestion
<script src="js/main.js"></script>
```